scripts/solution-manager.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
+set -e
+set -u
+set -o pipefail
+
+SOLUTION_CONFIG="/etc/metalk8s/solutions.yaml"
+VERBOSE=${VERBOSE:-0}
+LOGFILE="/var/log/metalk8s-solution-manager.log"
+SALT_CALL=${SALT_CALL:-salt-call}
+CRICTL=${CRICTL:-crictl}
+SALT=""
+SALTENV=""
+PILLAR=""
+
+
+_usage() {
+    echo "solution-manager.sh [options]"
+    echo "Options:"
+    echo "-s/--solution <solution path>:   Path to solution ISO"
+    echo "-n/--name <solution name>:       Solution name"
+    echo "--version <solution version>:    Version fo the solution"
+    echo "-l/--log-file <logfile_path>:    Path to log file"
+    echo "-v/--verbose:                    Run in verbose mode"
+}
+
+SOLUTIONS=()
+while (( "$#" )); do
+  case "$1" in
+    -s|--solution)
+      SOLUTIONS+=("$2")
+      shift 2
+      ;;
+    -v|--verbose)
+      VERBOSE=1
+      shift
+      ;;
+    -l|--log-file)
+      LOGFILE="$2"
+      [ -z "$DEBUG" ] && DEBUG="on"
+      shift 2
+      ;;
```

Comment:
`--name` and `--version` seems not supported :)

Reply:
This script was left aside when working on the demo, since it doesn't bring much value. Adding a line to `/etc/metalk8s/solutions.yaml` is quite straightforward, and this script can't do much more. Maybe we could remove it altogether, not sure though.

Reply:
But this script should also configure and mount the solutions ISOs ? No ?

Reply:
Do almost same thing as the `iso-manager.sh` script ?

Reply:
Mmh, maybe. To me, a good workflow would be:

1- Edit the config file to match your needs
2- Run the _good_ command: `kubectl metalk8s import-solutions`

But maybe we prefer a flow where adding/removing a Solution version is the unit of work (and we run the `import-solutions` on each addition/removal.

Reply:
This should be dont as a proper kubectl plugin, it is more of a helper script for the moment